app/src/app/framework/object/components/object-references/object-references.component.scss

```diff
@@ -0,0 +1,21 @@
+.scrollable .nav-tabs {
+  overflow-x: auto;
+  overflow-y:hidden;
+  flex-wrap: nowrap;
+  scrollbar-width: thin;
+
+  &::-webkit-scrollbar {
+    height: 5px;
+  }
+}
+
+.container {
```

Comment:
Try not to overwrite bootstrap classes. Just create a new class that inherits the behavior. In this case it works, because the view encapsulation of the component is set to default, but it's a bit messy.